services/QuillLMS/spec/services/demo/create_admin_report_spec.rb

```diff
@@ -47,13 +47,14 @@
     end
   end
 
-  it 'should create every teacher account from the data hash, associated with the correct school' do
+  it 'should create every teacher account from the data hash, associated with the correct school and with a login record' do
     schools_and_teachers = subject.data.map { |d| { 'School' => d['School'], 'Teacher' => d['Teacher'] } }.uniq
     schools_and_teachers.each do |row|
       teacher = User.find_by_name(row['Teacher'])
       school = School.find_by_name(row['School'])
       expect(SchoolsUsers.find_by(school: school, user: teacher)).to be
       expect(UserSubscription.find_by(user: teacher, subscription: subscription)).to be
+      expect(UserLogin.find_by(user: teacher)).to be
```

Comment:
I would use exists here over find_by since nothing is done with the record:
`UserLogin.exists?(user: teacher).to be true`